src/test/java/PackagelessClassTest.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 and Eclipse Distribution License v. 1.0
+ * which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * Contributors:
+ *     Ehsan Zaery Moghaddam (zaerymoghaddam@gmail.com) - initial implementation
+ ******************************************************************************/
+import org.junit.Test;
+
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests the serialization/deserialization of a class that has no package
```

Comment:
We are always putting a dot "." at the end of the sentence in javadocs. I would like it to be consistent.